v19.2/update.md

```diff
@@ -476,6 +502,59 @@ IDs:
 
 </section>
 
+### Update on a specific index
+
+Suppose that you create a multi-column index on the `users` table with the `name` and `city` columns.
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE INDEX ON users (name, city);
+~~~
+
+Now suppose you want to update a couple rows in the table, based on their contents. You can use the [`EXPLAIN (OPT)`](explain.html#opt-option) command to see how the [cost-based optimizer](cost-based-optimizer.html) decides to perform the `UPDATE` statement:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> EXPLAIN (opt) UPDATE users SET name='Michael Brown (there are two)' WHERE name='Michael Brown';
+~~~
+
+~~~
+                                      text
++-------------------------------------------------------------------------------+
+  update users
+   └── project
+        ├── index-join users
+        │    └── scan users@users_name_city_idx
+        │         └── constraint: /8/7/6: [/'Michael Brown' - /'Michael Brown']
+        └── projections
+             └── const: 'Michael Brown (there are two)'
+(7 rows)
+~~~
+
+The output of the `EXPLAIN` statement shows that the optimizer scans the newly-created `users_name_city_idx` index when performing the update. This makes sense, as you are performing an update based on the `name` column. If you want to perform the update using the primary index instead, to compare plans and tune performance, you can provide an index hint (i.e. force the index selection) to use a different index of the `users` table:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> EXPLAIN (opt) UPDATE users@primary SET name='Michael Brown (there are two)' WHERE name='Michael Brown';
+~~~
+
+~~~
+                          text
++-------------------------------------------------------+
+  update users
+   └── project
+        ├── select
+        │    ├── scan users
+        │    │    └── flags: force-index=primary
+        │    └── filters
+        │         └── name = 'Michael Brown'
+        └── projections
+             └── const: 'Michael Brown (there are two)'
+(9 rows)
+~~~
+
+As you can see from the output of the `EXPLAIN` statement, the plan changed quite a bit when you forced the index selection.
```

Comment:
It changed, but is this better or worse than before?  It's not at all clear to me as a non-expert at reading EXPLAINs.  The narrative structure/flow of this seems to suggest to me that the second result is something "better" (maybe?).  But the text content seems to be saying "it depends on which of these is better, but we ain't saying".  If the latter is the case I think we should be more explicit about that.